src/tzkt/tzkt_block_api.py

```diff
@@ -0,0 +1,24 @@
+from tzkt.tzkt_api import TzKTApi, TzKTApiError
+from api.block_api import BlockApi
+
+
+class TzKTBlockApiImpl(BlockApi):
+
+    def __init__(self, nw, verbose=False, base_url=None):
+        super(TzKTBlockApiImpl, self).__init__(nw)
+        if base_url is None:
+            self.api = TzKTApi.from_network(nw['NAME'].lower(), verbose=verbose)
```

Comment:
In the current implementation of the TRD, we fix the development network name to Alphanet, but it is meant to be the current active test network. I think tzkt expects the name of the active test network and not alphanet, right?